repository/src/main/java/org/apache/atlas/repository/store/graph/v2/preprocessor/ConnectionPreProcessor.java

```diff
@@ -17,6 +17,8 @@
  */
```

Comment:
Following scenario failing:
- create a connection with only “adminRoles”
- This resulted into adding current user into “adminUsers”
- With same user, to above create request I just added new “description”, update the connection
- Now creator user is removed from Keycloak role but it present in “adminUsers” on Connection
This impacts creator user not having access to Connection at all